backend/src/forms/song_form.py

```diff
@@ -0,0 +1,10 @@
+from flask_wtf import FlaskForm  # pyright: ignore
+from wtforms import StringField
+from wtforms.validators import DataRequired
+
+
+class SongForm(FlaskForm):
+    name = StringField("song-name", validators=[DataRequired()])
+    genre = StringField("genre")
+    thumb_url = StringField("thumbnail-img")
+    song_ref = StringField("song-ref", validators=[DataRequired()])
```

Comment:
My and joec's routes both accept data as json but flask-form takes data as urlencoded, the frontend can accommodate either way but it would be nice to be consistent
flask-forms do have a benefit of easier to write validations though, can keep it if you feel strongly

Reply:
I'm a little lost by this.  What specifically would I change?

Reply:
you can see how its done in https://github.com/ultrabear/soundclone/pull/18/files#diff-17874ced4efe9b4f52223ad5e8fed0683c16f5ea1e78851125155f5e3d4bf7a1R46
basically, using request.json to get the body as a json object and doing form checks on it

Reply:
ok, I was in the right place!  I was just looking at your comments example of this.  If it's ok with you I will leave it as FlaskForm and use those validators for now.  Seems like a less important thing to change right now unless you foresee major issues/bugs coming from doing it this way

Reply:
also I don't see where you're validating the comment text to make sure that it's not an empty string?  Or were you going to just add that later/not check for that at all? I guess it's conceivable someone could leave a blank comment.  Just want to make sure I'm not missing something

Reply:
yes i forgot to add validations